HTTP_200/static/js/main.js

```diff
@@ -901,3 +901,20 @@ else{
 
 });
 
+
+function ValidateBranch(){
+
+  var course = document.getElementById("course").value
```

Comment:
Either use jquery like this `var course = $("#course").val();` or put semicolon at the end.